src/main/java/gov/usgs/earthquake/distribution/WebSocketListener.java

```diff
@@ -0,0 +1,8 @@
+package gov.usgs.earthquake.distribution;
+
+/**
+ * Allows overridden onMessage for different behavior of WebSocketClient onMessage
+ */
+public interface WebSocketListener {
+  public void onMessage(String message);
```

Comment:
Done. Need to discuss more whether the WebSocketNotificationReceiver needs to use these methods or not.

Reply:
Maybe a reconnect timer or similar in case an initial reconnect fails, it could try every minute until it connects

Reply:
Implemented a rudimentary timer: on a reconnect failure, the listener waits 60 seconds then tries to connect again. This will continue until the client is shut down, or the thread is interrupted.